web/controllers/api/search_controller.ex

```diff
@@ -5,8 +5,13 @@ defmodule Constable.Api.SearchController do
 
   alias Constable.Announcement
 
-  def create(conn, %{"query" => search_terms}) do
-    announcements = Announcement.search(search_terms) |> Repo.all
+  def create(conn, params = %{"query" => search_terms}) do
+    excludes = params["exclude_interests"] || []
+    announcements = 
+      search_terms
+      |> Announcement.search
+      |> Announcement.without_interests(excludes)
```

Comment:
`without_interests` seems like it could be stripping interests from the announcement. I think the wording in the param might make sense as the function name instead. Something like `Announcement.excluding_interests`? Not a big deal either way though. Just a thought